modules/updating-eus-to-eus-upgrade-console.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * updating/preparing-eus-eus-upgrade.adoc
+
+:_content-type: PROCEDURE
+[id="updating-eus-to-eus-upgrade-console_{context}"]
+= EUS-to-EUS upgrade using the web console
+
+.Prerequisites
+
+* Have access to the web console as a user with `admin` privileges.
+
+.Procedure
+
+. Upgrade any OLM Operators to versions that are compatible with both versions you are updating to. You can find these steps on the link:https://docs.openshift.com/container-platform/4.11/operators/admin/olm-upgrading-operators.html#olm-changing-update-channel_olm-upgrading-operators[Preparing for an Operator update] page.
+
+. Verify that all machine config pools display a status of `UPDATED` and that no machine config pool displays a status of `UPDATING`.
```

Comment:
it looks like we are expecting `Up to date` status here rather than `UPDATED`
![Screenshot 2023-03-08 at 1 34 15 PM](https://user-images.githubusercontent.com/12692381/223628023-0f6b1daf-8096-43d9-a04c-63819400cb27.png)

Reply:
Good call :) ty!

Reply:
if the machine config pools are not up to date, does it say "updating" instead?

Reply:
Going to go with the most recent UI docs :)